### ISSUE TITLE
Add the ability to use DataVolume as source

### ISSUE THREAD

**Issue by ravidbro** [kubevirt/containerized-data-importer#1365]
/kind enhancement

**What happened**:
When we want to create DataVolume from another DataVolume (clone) we have only the option to select PVC as the source, but 
if I reference a PVC of DataVolume which still importing/cloning, the clone will start immediately and will clone partial data.

**What you expected to happen**:
To have a source from type DataVolume that will wait for the source DataVolume to be ready before starting to clone the PVC of the DV.

**Comment by codificat** [comment]
Is this a duplicate of #1309 maybe?

**Comment by awels** [comment]
It is, and #1400 'Fixes' it because that PR now verifies that if you specify a PVC owned by a DV, that it will wait for the PVC to be fully populated as indicated by the DV before cloning. So essentially there is no need for a separate DV source anymore.
